examples/readme.md

```diff
@@ -0,0 +1,151 @@
+NNI offers a variety of features to support different search space modes, different training platforms, different tuners and different deeplearning frameworks. The goal of this tutorial is to summarize the different features that make it easy for users to start the task that suits them.
+
+We have selected the following common aspects:
```

Comment:
You can add a navigation bar that can jump quickly to the specific part from here. For example:
* different search space modes: "api", "annotation"
* different training platforms: "local", "remote", "pai", "kubeflow"
* different tuners: "tpe", "batch-tune", "hyperband"
* ..........